File: mpcCommand.py
import sys
import h5py

numZones = int(sys.argv[1])

mpcCommands = h5py.File('mpcOutput.hdf5','r')
output = mpcCommands['output'][:][0]
outputString = ''
for zone in range(numZones):
    outputString = outputString + str(output[2*zone]) + ' ' + str(output[2*zone + 1]) + ' '
outputString = outputString[0:-1]
sys.stdout.write(outputString)

File: mpcCommand.jl
using HDF5

numZones = parse(Int, ARGS[1]);

output = h5read("mpcOutput.hdf5", "output");
outputString = "";

for zone = 0:numZones-1
    outputString = string(outputString, output[1 + zone*2], " ", output[2 + zone*2], " ");
end

outputString = outputString[1:end-1];
write(STDOUT, outputString);
